orcid-web/src/main/java/org/orcid/frontend/web/controllers/AdminController.java

```diff
@@ -496,6 +493,67 @@ public Map<String, String> findIdByEmailHelper(String csvEmails) {
         return emailManager.findOricdIdsByCommaSeparatedEmails(csvEmails);
     }
 
+    @RequestMapping(value = "/add-email-to-record", method = RequestMethod.POST)
+    public @ResponseBody ProfileDetails addEmailToRecord(HttpServletRequest serverRequest, HttpServletResponse response, @RequestBody ProfileDetails profileDetails)
+            throws IllegalAccessException {
+        profileDetails.setErrors(new ArrayList<String>());
```

Comment:
@leomendoza123 the first line for any function in the AdminController should be: 

```
isAdmin(serverRequest, response);
```

Because that will verify that the credentials of the current user allow them to call that function